pcdsdevices/pseudopos.py

```diff
@@ -39,6 +39,53 @@ def __init__(self, prefix='', parent=None, **kwargs):
             prefix = f'{parent.prefix}:{attr_name}'
 
         super().__init__(prefix=prefix, parent=parent, **kwargs)
+        self._verbose_name = verbose_name
+
+    # TODO: this implementation might be totally crazy....
+    @property
+    def calculated_dial_pos(self):
```

Comment:
This  implementation here does not really work for all scenarios... so probably I took the wrong approach here. 
Works fine for `xcs_lxt_fast.delay` for example, but not for `xcs_ccm.calc.energy` :
`xcs_ccm.calc.energy` - has two `real_positioners`, and does real_positioners don't have a `dial_position`.

Reply:
If there is no dial position I think it is okay to assume that the dial position is the same as the normal position. This is actually the case for the ccm, it doesn't have any sort of user offsets that I'm aware of.

I think it may be possible to generalize to arbitrary numbers of real positioners by passing them all into inverse as a RealPositionTuple or whatever it's called

Reply:
Ok that's good then, I'll assume `dial_pos` = normal position if some objects don't have that - Thank you

As for the second one - I need to do more exploration here, I think the challenge is that I need the `dial_position` of those `real_positioners`  in the `inverse` method not the actual `normal position`

Reply:
Ohh I think I get it now, (maybe). I would just pass into `inverse` multiple dial_positions in my case:
```
self.parent.RealPosition(*dial_pos))
```
where `dial_pos` could be for example `[13.84, 0.0]`

This will now give me 2 calculated dial position - how would I go about choosing which one to use in the `format_status_info`? would I display both of them?

Reply:
I think what happens here is that you'll get out some number of "dial pseudo positions" or whatever we want to call them, in the psuedo tuple, with reasonable names. And then you can pick the correct one for the pseudo single hopefully.